README.md

```diff
@@ -87,6 +87,64 @@ Start the containers using docker compose:
 docker-compose up
 ```
 
+### Testing
+
+#### Running the standard test-suite
+
+running locally, in the `osmaxx-py` directory (using a virtualenv):
+
+```shell
+DJANGO_SETTINGS_MODULE=config.settings.test ./manage.py test
+```
+
+*Notice*: Since the ampq Service is not available, you'll get a 4 failing tests. Therefore it is recommended to
+run them in a docker container, as described below.
+
+Using docker and docker-compose:
+ 
+```shell
+docker-compose run --rm webappdev bash -c 'DJANGO_SETTINGS_MODULE=config.settings.test ./manage.py test'
+```
+
+#### Running the test-suite and integration tests
+
+There's a script to facilitate testing including some docker integration tests:
+
+```shell
+./test.sh
+```
+
+#### Running end to end tests (e2e)
+
+**Warnings**: 
```

Comment:
> _Warnings_ should be a subsection is what you mean, right?

Yes, and so should 'Prerequisites'.

> We could put the entire documentation on testing into a separate file, and have a link to there

Note, there's already [section 'Run tests' in `docs/project-development-environment.md`](https://github.com/geometalab/osmaxx/blob/e7f2dac9d737062dba5c668ffa0936a39fae72c7/docs/project-development-environment.md#run-tests) and [section 'Run tests' in `docs/useful-commands.md`](https://github.com/geometalab/osmaxx/blob/e7f2dac9d737062dba5c668ffa0936a39fae72c7/docs/useful-commands.md#run-tests). Probably their content and the above and howto coverage should go together in a separate file. Where appropriate in the other files, we can link to that file or its sections.